projects/components/src/quick-search/quick-search.component.ts

```diff
@@ -36,7 +37,7 @@ interface GroupedSearchSections {
     subSections: SearchSection[];
 }
 
-interface SearchSection {
+export interface SearchSection {
```

Comment:
Should these interfaces that are now exported be defined by the service? The service is the source of truth now?